### PR TITLE
support for f5 bigip

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 1]
```diff
@@ -6679,3 +6679,38 @@ display.page.search.mode = fast
...
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 10 * * * *
+description = A not blocked attack events from F5 BIGIP ASM \
+\
+Data Collection - Splunk Add-on for F5 BIG-IP.
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:6696]: Add addon splunk base link if we have.

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -6679,3 +6679,38 @@ display.page.search.mode = fast
...
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_f5_bigip_asm` ip_client="*" dest_ip="*" severity IN ("critical","high","medium") enforcement_action!=block attack_type!="JSON Parser Attack" \
+| eval cyences_severity = if(severity="informational", "info", severity) \
+| table _time cyences_severity severity ip_client src_port dest_ip dest_port manage_ip_addr x_fwd_hdr_val attack_type enforcement_action blocking_exception_reason client_type credential_stuffing_lookup_result device_id enforced_by geo_info http_class ip_addr_intelli ip_route_domain login_result method mobile_application_name mobile_application_version policy_apply_date policy_name protocol protocol_info req_status resp_code route_domain sig_ids sig_names sub_violates threat_campaign_names unit_host uri username violate_details violate_rate violations virus_name  is_trunct \
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:6706]: severity field is not required as we have cyences_severity

Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/savedsearches.conf:6706]: use cs_human_readable_time_format macro for _time

File: cyences_app_for_splunk/default/data/ui/views/cs_f5_bigip_asm.xml [hunk 3]
```diff
@@ -0,0 +1,242 @@
+<form version="1.1" theme="dark">
+  <label>F5 BIGIP ASM</label>
+  <fieldset submitButton="false">
+    <input type="time" token="timerange">
+      <label></label>
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/data/ui/views/cs_f5_bigip_asm.xml:5]: label = Time Range

File: cyences_app_for_splunk/default/data/ui/views/cs_f5_bigip_asm.xml [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <table>
+        <title>Attacks by IP</title>
+        <search>
+          <query>`cs_f5_asm`  attack_type!="N/A"  $tkn_src_ip$ $tkn_dest_ip$ $tkn_severity$
+| table _time ip_client src_port dest_ip dest_port manage_ip_addr x_fwd_hdr_val attack_type enforcement_action blocking_exception_reason client_type credential_stuffing_lookup_result device_id enforced_by geo_info http_class ip_addr_intelli ip_route_domain  login_result method mobile_application_name mobile_application_version policy_apply_date policy_name protocol protocol_info req_status resp_code route_domain severity sig_ids sig_names sub_violates threat_campaign_names unit_host uri username violate_details violate_rate violations virus_name  is_trunct
```
Comment (hardikhdholariya) [inline cyences_app_for_splunk/default/data/ui/views/cs_f5_bigip_asm.xml:227]: we don't need all these fields for upcoming stats command.

Comment (mahirchavda) [reply cyences_app_for_splunk/default/data/ui/views/cs_f5_bigip_asm.xml:227]: It is useful when we want to debug something by opening in the search query

**Review by hardikhdholariya — APPROVED** [review]

**Pull request merged**